README.adoc

```diff
@@ -20,7 +20,10 @@ $ docker --version
 $ docker-compose --version
 ```
 
-Install via curl
-```
-$ curl -L http://bit.ly/graviteeiodemo | bash
+Launch demo :
+
+Go to environments/demo/ and type :
+
 ```
+make run
```

Comment:
What about Windows users ?
Why the previous command was not sufficient and has been removed for this 'build' one ?